ansible/module_utils/port_utils.py

```diff
@@ -198,6 +198,11 @@ def get_port_alias_to_name_map(hwsku, asic_id=None):
         else:
             for i in range(1,9):
                 port_alias_to_name_map["Ethernet1/%d" % i] = "Ethernet%d" % ((i - 1) * 4)
+    elif hwsku == "B6510-48VS8CQ":
+        for i in range(1,48):
+            port_alias_to_name_map["twentyfiveGigE0/%d" % i] = "Ethernet%d" % i
+        for i in range(49,56):
+            port_alias_to_name_map["hundredGigE0/%d" % (i-48)] = "Ethernet%d" % i
```

Comment:
Can you remove the above change from this PR? Looks like #3406 has already contained it?